.hashibot.hcl

```diff
@@ -11,6 +11,20 @@ poll "closed_issue_locker" "locker" {
   EOF
 }
 
+poll "stale_issue_closer" "closer" {
+    schedule = "0 50 12 * * *"
+    no_reply_in_last = "2160h" # 90 days
+    max_issues = 500
+    sleep_between_issues = "5s"
+    created_after = "2019-11-11"
```

Comment:
I'm not certain on why we only want to consider issues after today, but that is outside the context of the review, I'm sure this was very intentional.

Reply:
There are some concerns about inflicting this on people who might have other expectations -- I'll ping the rest of the team to make sure we couldn't go earlier.